packages/document/docs/en/guide/basic/custom-page.mdx

```diff
@@ -167,6 +167,19 @@ import PageType from '@en/fragments/page-type';
 
 <PageType />
 
+### Using frontmatter to generate page metadata for better SEO
```

Comment:
The `Customizing header tag` section is missing?

Reply:
Waiting for husky 👀 hold up

Reply:
My bad, last push was to another branch. Sorry closing this one now :(